spark-connector/build.gradle.kts

```diff
@@ -19,21 +21,28 @@ val kyuubiVersion: String = libs.versions.kyuubi.get()
 val scalaJava8CompatVersion: String = libs.versions.scala.java.compat.get()
 
 dependencies {
-  implementation(project(":api"))
   implementation(project(":clients:client-java-runtime", configuration = "shadow"))
-  implementation(project(":common"))
-  implementation(libs.bundles.log4j)
   implementation(libs.guava)
   implementation("org.apache.iceberg:iceberg-spark-runtime-3.4_$scalaVersion:$icebergVersion")
   implementation("org.apache.kyuubi:kyuubi-spark-connector-hive_$scalaVersion:$kyuubiVersion")
-  implementation("org.apache.spark:spark-catalyst_$scalaVersion:$sparkVersion")
-  implementation("org.apache.spark:spark-sql_$scalaVersion:$sparkVersion")
   implementation("org.scala-lang.modules:scala-java8-compat_$scalaVersion:$scalaJava8CompatVersion")
```

Comment:
Will it include scala jar in the package? This may cause conflict with Spark.

Reply:
will make it compileonly

Reply:
done